hnginternship5/ziki
# 404 page optimised to fit new adjustments

What does this PR do?
• Adjusted the 404 pageDescription of Task to be completed?
• optimised page to fit new update on sidebar
How should this be manually tested?
• Open the site on your browser
• Type localhost:8000/404
• Check the alignment.What pages did you work on?
• Page 404.html
Screenshots (mandatory)
--

![Screen Shot 2019-05-04 at 2 32 01 AM](https://user-images.githubusercontent.com/38491485/57177292-c738a100-6e5a-11e9-87ea-df964c6293c4.png)
